packages/editor/src/components/EditorContainer.tsx

```diff
@@ -76,6 +78,18 @@ export const DockContainer = ({ children, id = 'editor-dock', dividerAlpha = 0 }
   )
 }
 
+const onEditorWarning = (warning) => {
+  console.warn(warning)
```

Comment:
this console warning can be removed since notification message has the same text

Reply:
this is so that the warning is logged in the console, and we can extract the logs for other purposes later, and the warning stays in there